src/widgets/popOver.test.js

```diff
@@ -0,0 +1,248 @@
+			my: 'left',
+			at: 'left bottom',
+			collision: 'flipfit'
+		};
+
+		setup(undefined, options);
+
+		pow.position();
+		clock.tick();
+
+		expect(spy.position).toHaveBeenCalled();
+		expect(spy.element.position).toHaveBeenCalled();
+
+		expect(spy.element.position).toHaveBeenCalledWith({
+			my: options.my,
+			at: options.at,
+			of: options.target,
+			collision: options.collision
+		});
+	});
+
+	it('can destroy itself', () => {
+		expect(pow).toBeTruthy();
+		pow._destroy();
+
+		const popWidget = pop.data('formBuilderPopOver');
+		expect(popWidget).toNotExist();
+	});
+
+	it('is positioned on window resize when showing', () => {
+		$(window).trigger('resize');
+		expect(spy.position).toNotHaveBeenCalled();
+
+		pow.show();
+		clock.tick();
+		
+		$(window).trigger('resize');
+		expect(pow.position).toHaveBeenCalled();
+	});
+
+	it('is hidden when clicking outside of its target and itself', () => {
+		pow.show();
+		clock.tick();
+
+		target.trigger('mousedown');
+		clock.tick();
+		expect(spy.hide).toNotHaveBeenCalled();
+
+		target.children('h1').trigger('mousedown');
+		clock.tick();
+		expect(spy.hide).toNotHaveBeenCalled();
+
+		pop.trigger('mousedown');
+		clock.tick();
+
+		expect(spy.hide).toNotHaveBeenCalled();
+
+		testContainer.trigger('mousedown');
+		clock.tick();
+
+		expect(spy.hide).toHaveBeenCalled();
+	});
+
+	describe('is appended to a new element', () => {
+		it('that is the body be default', () => {
+			expect(pop.parent().is('body')).toBe(true);
+		});
+
+		it('that can be set and changed with a function call', () => {
+			expect(pop.parent().is('body')).toBe(true);
+			expect(pop.parent().is(testContainer)).toBe(false);
+			pow.setAppendTo(testContainer);
+			expect(pop.parent().is('body')).toBe(false);
+			expect(pop.parent().is(testContainer)).toBe(true);
+		});
+
+		it('that can be set with a widget option', () => {
+			pow._destroy();
+			pop.remove();
+
+			const options = {
+				appendTo: testContainer
+			};
+
+			setup(undefined, options);
+
+			expect(pop.parent().is('body')).toBe(false);
+			expect(pop.parent().is(testContainer)).toBe(true);
+		});
+	});
+
+	describe('can have a title', () => {
+		it('that is hidden and empty by default', () => {
+			const title = pop.children('.tooltip-title');
+			expect(title.length).toBe(1, 'Title element was found');
+			expect(title[0].style._values.display).toBe('none');
+			expect(title.text()).toBe('');
+		});
+
+		it('that can be set with an attribute', () => {
+			setup('<div title="Some Title!"><p>Some content</p><br><h4>And some more content</h4></div>');
+
+			const title = pop.children('.tooltip-title');
+
+			expect(title.length).toBe(1, 'Title element was found');
+			expect(title[0].style._values.display).toNotExist('Title should be visible');
+			expect(title.text()).toBe('Some Title!');
+		});
+	});
+
+	describe('can be toggled', () => {
+		it('from shown to hidden', () => {
+			pow.show();
+
+			clock.tick();
+
+			// expect.spyOn(pow, 'hide');
```

Comment:
Remove commented code or un-comment it.  Set the test to skip if it doesn't pass.